futurex_openedx_extensions/helpers/tenants.py

```diff
@@ -97,6 +97,7 @@ def get_all_tenants_info() -> Dict[str, str | dict | List[int]]:
                     (tenant['lms_configs'].get('platform_name') or '').strip(),
                 ], default=''),
                 'logo_image_url': (tenant['lms_configs'].get('logo_image_url') or '').strip(),
+                'nafath_enabled': str(tenant['lms_configs'].get('nafath_enabled', '')).lower() == 'true',
```

Comment:
having the flag here is a great idea. But we do not have `nafath_enabled` entry in the configs. You should add a new function that collects the list of all sites having Nafath enabled. To do that; I believe the best way is to filter on the Entry-ID for all SAML provider configs. See `admin/third_party_auth/samlproviderconfig/` (Declare something like `FX_NAFATH_ENTRY_ID` in settings and use it for the filter)

After that, use the list to enable/disable your flag here